journal/journal.go

```diff
@@ -98,12 +104,7 @@ func Send(message string, priority Priority, vars map[string]string) error {
 
 		rights := syscall.UnixRights(int(file.Fd()))
 
-		/* this connection should always be a UnixConn, but better safe than sorry */
-		unixConn, ok := conn.(*net.UnixConn)
-		if !ok {
-			return journalError("can't send file through non-Unix connection")
-		}
-		unixConn.WriteMsgUnix([]byte{}, rights, nil)
+		conn.WriteMsgUnix([]byte{}, rights, socketAddr)
```

Comment:
Your comment comes just in time, I didn't have time to address the few remaining actions and I wanted to revive the merge request ;-)

I was also wondering if we should check the error or not (as this is an unconnected socket) or silently fail. Maybe it's a good idea to return a `journalError`.

At this step, I see several options :

- ping this thread on golang-nuts, as suggested by @philips : https://groups.google.com/forum/#!msg/golang-nuts/PyOeimgyvFw/VHNCxxEbDP0J
- provide a custom function to write the message, as suggested by @s-urbaniak : s-urbaniak@a715600 (maybe in another merge request)

Any advice ?